modules/git/commit.go

```diff
@@ -215,8 +215,8 @@ func (c *Commit) CommitsBeforeUntil(commitID string) (*list.List, error) {
 }
 
 // SearchCommits returns the commits match the keyword before current revision
-func (c *Commit) SearchCommits(keyword string, all bool) (*list.List, error) {
-	return c.repo.searchCommits(c.ID, keyword, all)
+func (c *Commit) SearchCommits(keywords, authors, committers []string, after, before string, all bool) (*list.List, error) {
```

Comment:
At this point I'd probably say it's worth creating an options struct.

Reply:
Done.